CHANGELOG.md

```diff
@@ -145,7 +146,7 @@ Now connector field is mandatory in helm chart is mantatory.
 ## [2.2.1](https://github.com/idealista/prom2teams/tree/2.2.1)
 [Full Changelog](https://github.com/idealista/prom2teams/compare/2.1.2...2.2.1)
 ## Added
-- *[#80](https://github.com/idealista/prom2teams/pull/79) Add the possibility of group alarms by alertname* @manuhortet
```

Comment:
Should we also update the changelog of already released versions?

Reply:
I would rather use the same name to refer to the same concept. In that sentence "alarms" and "alerts" are mixed.